### ISSUE TITLE
Getting wails.localhost refused to connect at startup

### ISSUE THREAD

**Issue by kishorkumar287** [wailsapp/wails#2526]
### Description

I am receiving wails.localhost refused to connect page still my app loads and after few seconds it works perfectly. This issue is reproducuble from before three days only. Before there was no error page like this. Please suggest me what to do. Attached a screenshot for your reference.

### To Reproduce

build a wails application and launch it, after that you can see an error for few seconds.

### Expected behaviour

There should be no error page in the begining of the window.

### Screenshots

![image](https://user-images.githubusercontent.com/60978027/227474088-ad9ee458-edc4-43b2-8d85-c70038d8f78b.png)

### System Details

```shell
DEB | Using go webview2loader
Wails CLI v2.3.1

Scanning system - Please wait (this may take a long time)...Done.

# System

OS           | Windows 10 Home Single Language
Version      | 2009 (Build: 22621)
ID           | 22H2
Go Version   | go1.19.4
Platform     | windows
Architecture | amd64

# Wails

Version | v2.3.1

# Dependencies

Dependency | Package Name | Status    | Version      
WebView2   | N/A          | Installed | 111.0.1661.51
npm        | N/A          | Installed | 8.1.2
*upx       | N/A          | Available | 
*nsis      | N/A          | Available | 
* - Optional Dependency

# Diagnosis

Your system is ready for Wails development!
Optional package(s) installation details:
  - upx : Available at https://upx.github.io/
  - nsis : More info at https://wails.io/docs/guides/windows-installer/

 ♥   If Wails is useful to you or your company, please consider sponsoring the project:
https://github.com/sponsors/leaanthony
```

**Comment by stffabi** [comment]
Thanks for using Wails and reporting this issue.

This issue has already been fixed with the release of v2.4.1.

**Comment by stffabi** [comment]
Duplicate of #2521
